drizzle/migrations/0003_clerk_auth_sync.sql

```
@@ -0,0 +1,398 @@
+-- Locations
+DROP POLICY IF EXISTS "Users can manage their own locations" ON public.locations;
+CREATE POLICY "Users can manage their own locations" ON public.locations
+  FOR ALL USING (user_id = auth.uid() OR public.is_clerk_user(user_id));
+
+-- Visualizations
+DROP POLICY IF EXISTS "Users can manage their own visualizations" ON public.visualizations;
+CREATE POLICY "Users can manage their own visualizations" ON public.visualizations
+  FOR ALL USING (user_id = auth.uid() OR public.is_clerk_user(user_id));
+
+-- Chat participants
+DROP POLICY IF EXISTS "Only owners can insert participants" ON public.chat_participants;
```

SQLFluff (4.2.2)

[error] Do not use special characters in identifiers.

(RF05)